code-challenges.test.js

```diff
@@ -14,46 +14,153 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const numbersArray1 = [6, 7, 8, 9, 10]
+// const numbersArray1 = [6, 7, 8, 9, 10]
 // Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
+// const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
 
 // b) Create the function that makes the test pass.
 
+// ===================================== CHALLENGE #1 =================================================
+
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+describe("isDivisibleByThree", () => {
+  it("the function will take in an object and check if te number inside is divisible by 3", () => {
+    const object1 = { number: 15 };
+    // Expected output: "15 is divisible by three"
+    const object2 = { number: 0 };
+    // Expected output: "0 is divisible by three"
+    const object3 = { number: -7 };
+    // Expected output: "-7 is not divisible by three"
+    expect(isDivisibleByThree({ number: 15 })).toEqual(
```

Comment:
Since your test variables are already declared, you can pass those into your function inside your expect statement.